radio/src/strhelpers.cpp

```diff
@@ -294,7 +294,7 @@ char * getSwitchString(char * dest, swsrc_t idx)
 #endif
 
   else if (idx <= SWSRC_LAST_LOGICAL_SWITCH) {
-    strAppendStringWithIndex(s, "L", idx-SWSRC_FIRST_LOGICAL_SWITCH+1);
+    strAppendUnsigned(strAppend(s, "L"), abs(idx-SWSRC_FIRST_LOGICAL_SWITCH+1), 2);
```

Comment:
I used the call from `strAppendStringWithIndex()` as is, IMO brings less potential confusion in the future (pattern can be easily found). OK to change if you feel differently.